guides/RELEASE-PROCESS.md

```diff
@@ -18,7 +18,12 @@
       ```
       This script will install dependencies, build design system assets, and run the linter and tests. If everything passes, `lerna` will prompt you for the new version number, which should follow the [SemVer release format](#versioning). After the versions are updated, the script will create a tagged release commit (prefaced with `core-`), a branch (prefaced with `release-`) containing the release commit, and a zip for the Github release notes.
    1. The last step in the script will prompt you to continue with publishing to NPM. Proceed if everything is ready.
```

Comment:
I wonder if its less confusing to remove the `yarn release` command, and just break it into `yarn prepublish` and `yarn publish`. This way we can include this step: "visually check that release zip created from the release script (i.e. `cmsgov-design-system-2.0.0.tgz`) contains the folders and files" before publishing files

Reply:
That's a good suggestion, it'll help add another level of checking in breaking `yarn release` into `yarn prepublish` and `yarn publish`

Reply:
@bernardwang found that `prepublish` and `publish` are NPM life cycle events, so what do you think if we change the process to `yarn pre-release` and `yarn release`?

Reply:
Can we just use the life cycle events? So something like this:
```
    "prepublish": "./prepublish.sh",
    "publish": "yarn lerna publish from-git",
```
Then going forward in order to publish the release you can just run `yarn run publish` or `npm run publish`, which will run prepublish and publish lifecycle scripts.

Reply:
Actually, I guess if we use the lifecycle events it wont be possible to separate the 2 commands because the publish lifecycle script automatically runs the prepublish lifecycel. Still, I kind of like the simplicity of using the lifecycle events, and you can always  directly run the commands (i.e. run `yarn lerna publish from-git` instead of `npm run publish`)

Reply:
That would be great, but when I used `"prepublish": "./prepublish.sh"` , running `yarn install` ended up executing the prepublish script

Reply:
Looks like `yarn install` calls the `prepublish`, I'm thinking it's okay to stay with `pre-release` & `release` for now and consider the `prepublish` hook for another PR?

Reply:
@nichia I forgot to mention this, we should be using `prepublishOnly` instead of `prepublish`
Heres more background: https://docs.npmjs.com/misc/scripts

Reply:
I've changed package.json to have the `prepulishOnly` and `publish` script, however, getting an error message when I run `yarn publish`
```yarn publish
yarn publish v1.22.5
error Package marked as private, not publishing.```

Found this issue [issue about it](https://github.com/yarnpkg/yarn/issues/5334) but I cannot find where we specify `private:true` on the package.json.

Reply:
Sorry again for the back and forth, but I saw this comment and we (shouldnt be overriding the lifecycle scripts)[https://github.com/lerna/lerna/issues/1969#issuecomment-498694766]. Maybe its best to keep it to what you had, which is `prerelease` and `release`. Also, please also update the script to be called `prerelease.sh` as well